templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -113,13 +113,14 @@ contents:
     BRIDGE_METRIC="49"
```

Comment:
conflict: conflict: remove mentions of BRIDGE1 as 4.8 has no support for secondary bridge in configure-ovs.